src/libraries/System.IO/tests/BufferedStream/BufferedStreamTests.cs

```diff
@@ -12,6 +12,8 @@ namespace System.IO.Tests
 {
     public class BufferedStream_StreamAsync
     {
+        public static bool IsX64 { get; } = IntPtr.Size >= 8;
```

Comment:
This isn't necessary... you can use this:
https://github.com/dotnet/runtime/blob/57bfe474518ab5b7cfe6bf7424a79ce3af9d6657/src/libraries/System.Runtime.InteropServices/tests/System/Runtime/InteropServices/NFloatTests.cs#L43

Reply:
Fixed, thanks!